texts/guest_invitation.inc

```diff
@@ -6,7 +6,9 @@ We love the work you do and would be grateful to have you bring something to sha
 
 If you are interested, our CFP is live now: $CFP_link
 
-For DEF CON 29, things are still in a bit of a state of flux. However, we are moving forward on the assumption that the DEF CON hybrid event will take place. Similar to Safe Mode DEF CON 28, all talks and events are going to be pre-recorded to be able to stream online and play in-person. Keep an eye on the DEF CON website for updates!
+Also, this year we will be running a "Make Your Own Use" contest! We are looking for judges, would you be a judge? The concept of the contest is submissions can be made based on any project that worked on over the year, there may be 3 categories for a) repairing a closed device, b) adding a feature to a device that was never intended, c) bypassing DRM on devices to unlock their full potential -- if there are many submissions. We expect the judges to award points based on 1) how bent/changed/diferent/unintended it is 2) re-work quality 3) re-work ingenuity 4) weirdness (it is DEF CON).
```

Comment:
Suggest changing "would you be a judge" to "would you be interested in volunteering to be a judge"

Reply:
good note, thanks bunni